emcc.py

```diff
@@ -2660,7 +2660,7 @@ def get_full_import_name(name):
     # What you need to do is different depending on the kind of EH you use
     # (https://github.com/emscripten-core/emscripten/issues/17115).
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$getExceptionMessage', '$incrementExceptionRefcount', '$decrementExceptionRefcount']
-    settings.EXPORTED_FUNCTIONS += ['getExceptionMessage', '___get_exception_message']
+    settings.EXPORTED_FUNCTIONS += ['___get_exception_message']
```

Comment:
Maybe I'm mistaken, but I thought `EXPORTED_FUNCTIONS` is for C/C++ library functions that we want to bring in. And we have this `getExceptionMessage` in `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE`, which includes it in the js file. And `core.test_EXPORT_EXCEPTION_HANDLING_HELPERS` is passing without this, so apparently we don't need this..?

Reply:
EXPORTED_FUNCTIONS (IIUC) is also for JS library functions if you want them to be exported to the outside world.

DEFAULT_LIBRARY_FUNCS_TO_INCLUDE just includes them in the link, but it doesn't prevent from from DCE'd, or cause them to be exported.  IIUC.

Reply:
The `test_EXPORT_EXCEPTION_HANDLING_HELPERS` should probably be updated such it called `Module.getExceptionMessage()` from the outside (e.g. in `--extern-post-js`)... if we did that I think this change would cause it to fail.

Reply:
I tried to add a new test that calls `getExceptionMessage` but it doesn't seem to be easy.. I need to wrap `run()` with `try`-`catch` do have access to the exception object, but `--extern-post-js` allows me to only append things at the end.